Dockerfile.base

```diff
@@ -0,0 +1,6 @@
+# base image built from https://github.com/just-the-docs/just-the-docs/blob/main/Dockerfile
+FROM quay.io/hari_balagopal/just-the-docs:v1
```

Comment:
We can put it either in Move2Kube or Konveyor orgs in quay.

Reply:
sure, can you add it? don't have the perms